doc/source/conf.py

```diff
@@ -194,6 +194,14 @@
 \usepackage{etoolbox}
 \makeatletter
 \patchcmd\@item{{\@breaklabel} }{{\@breaklabel}}{}{}
+% Fix bug in expdlist's way of breaking the line after long item label
+\def\breaklabel{%
+    \def\@breaklabel{%
+        \leavevmode\par
+        % now a hack because Sphinx inserts \leavevmode after term node
+        \def\leavevmode{\def\leavevmode{\unhbox\voidb@x}}%
```

Comment:
Does this also have a guard against long oneliner items or multiparagraph items? Also is this scoped?

Reply:
The `\@breaklabel` is scoped from the fact that it is excuted in an item lable in a description list. Sphinx always add a `\leavevmode` (see [here](https://github.com/sphinx-doc/sphinx/blob/v1.7.2/sphinx/writers/latex.py#L1635)). The redefinition makes it inoperant. But the point is that if the item contents is a paragraph, the first character does not need a `\leavevmode`, it will start a paragraph. If the item contents is a list itself, we want to *not* start a paragraph which would case an extra blan line.

The issue arises because ``expdlist`` wants to force a line break in a case of wide item label. But the way it does it is bugger (cf TeX by Topic, ``17.2.2 Ending a paragraph with \hfill\break``, which is to what ``expdlist`` buggy code amount, using a `\\` (whihc by the way could cause problems in tables)). To achieve same goal I need the `\par`. This closes the paragraph doing the item label. But then a workaround has to be found for the `\leavevmode` inserted by Sphinx mark-up.

In all cases there is a `\leavevmode` and it is executed to recover its LaTeX meaning. So it will have correct meaning for what comes next.